pgcli/main.py

```diff
@@ -1626,6 +1634,25 @@ def cli(
 
         sys.exit(0)
 
+    if ping_database:
+        results = None
+        try:
+            results = list(pgcli.pgexecute.run("SELECT 1"))
+        except Exception:
+            # yup, really, anything here
+            ...
+
+        if results:
+            click.echo("PONG")
+            sys.exit(0)
+        else:
+            click.secho(
+                "Could not connect to the database. Please check that the database is running.",
+                err=True,
+                fg="red",
+            )
+            sys.exit(1)
```

Comment:
[style] The comment in the `except` block sounds a bit weird to me. I think that an `else` block would be clearer (and in fact shorter):

```suggestion
        except Exception:
            click.secho(
                "Could not connect to the database. Please check that the database is running.",
                err=True,
                fg="red",
            )
            sys.exit(1)
        else:
            click.echo("PONG")
            sys.exit(0)
```